src/test/testtool/luminizer/PostZaak/02/Input.xml

```diff
@@ -0,0 +1,10 @@
+<root>
+	<taaktype>https://localhost/api/taaktypes/69f5ead5-1d53-4398-ac69-a3ca31854125</taaktype>
+	<titel>Hoofdriool</titel>
```

Comment:
Also here the titel and taaktype don't refer to the ones in luminizer_zaaktypes.xml